apps/docs/src/.vitepress/components/IconLibraryItem.vue

```diff
@@ -17,16 +18,19 @@ const handleCopy = async () => {
   isCopied.value = true;
   setTimeout(() => (isCopied.value = false), 3000);
 };
+
+const tooltipText = computed(() => {
+  if (!isCopied.value) return props.icon.tooltipName;
+  return `${props.icon.tooltipName} - Import copied to clipboard!`;
```

Comment:
is it possible to keep the 2-row layout (= the import text on a line below the icon name) to prevent that the icon name jumps to the side when copying?

Reply:
I added the import text above the icon name because it looks prettier with the check icon :)